lms/djangoapps/support/static/support/js/views/enrollment.js

```diff
@@ -39,12 +39,7 @@
                     user: user,
                     enrollments: this.enrollments,
                     formatDate: function(date) {
-                        if (!date) {
-                            return 'N/A';
-                        }
-                        else {
-                            return moment(date).format('MM/DD/YYYY (H:MM UTC)');
-                        }
+                        return date ? moment.utc(date).format('lll z') : 'N/A';
```

Comment:
From a product perspective, I think this is fine - are there any open questions still? I would recommend also confirming with the reporters that their issue is resolved.